src/pages/admin/regulatory/forms/CreateRegulatoryForm.js

```diff
@@ -161,14 +161,14 @@ const CreateRegulatoryForm = props => {
                                <Grid container direction="row" spacing={8} justify="space-between">
                    
                                 <Grid item xs ={4}>
-                                  <Field name ={`contacts.${index}.name`}
+                                  <Field name ={`attachment.${index}.name`}
                                       label  = "Document Name"
                                       fullWidth
                                       component = {TextField}
                                       variant = "outlined"/>
                                </Grid> 
                                <Grid item xs = {3}>
-                                 <Field name = {`contacts.${index}.description`}
+                                 <Field name = {`attachment.${index}.description`}
```

Comment:
This is how you access attachment on map function `attachment.${index}.name` is similar to attachment[0].name or attachment.0.name
attachments.map ((attachment, index) =>